asllib/error.ml

```diff
@@ -168,6 +169,7 @@ let error_label = function
   | RecursionLimitReached -> "RecursionLimitReached"
   | EmptyConstraints -> "EmptyConstraints"
   | UnexpectedPendingConstrained -> "UnexpectedPendingConstrained"
+  | BadPrintType _ -> "BadPrintType"
```

Comment:
Consider adding a type for a clearer type message. If there is a list of expressions, it's going to be hard to tell which one caused the problem.